test-framework/common/src/main/java/io/quarkus/test/common/QuarkusTestResourceLifecycleManager.java

```diff
@@ -32,4 +32,14 @@
      */
     default void inject(Object testInstance) {
     }
+
+    /**
+     * If multiple Test Resources are specified,
+     * this control the order of which they will be executed.
+     * 
+     * @return The order to be executed. The bigger the number, it is executed later.
```

Comment:
Would probably be better phrased like so: `The larger the number, the later the Resource is invoked`